apps/nuxt3-ssr/interfaces/types.ts

```diff
@@ -128,7 +128,7 @@ interface ICollectionEventCategorySet {
   definition?: string;
 }
 
-interface INetwork {
+export interface INetwork {
   id: string;
```

Comment:
I do understand why you do it, but can we generate these from graphql? Otherwise maintaining these is a huge hassle as for every column added in data model we must add it in interface and in frontend, right?

Reply:
if you want to lock into a one to one mapping it could be generated , not sure if thats better, generating the details is also non trivial and maybe better to fond/join a opensource project for this.